.github/workflows/testing-dependency.yaml

```diff
@@ -158,3 +158,19 @@ jobs:
           index_name: '${{ needs.dependency-matrix-setup.outputs.index_name }}'
           PINECONE_API_KEY: '${{ secrets.PINECONE_API_KEY }}'
           urllib3_version: '${{ matrix.urllib3_version }}'
+
+  deps-cleanup:
```

Comment:
This is not really related to the config changes, but keeps us from getting values due to 20 index per-project limit.